actor/src/main/scala/com/evolutiongaming/akkaeffect/Act.scala

```diff
@@ -72,12 +80,20 @@ private[akkaeffect] object Act {
         val value = {
           class Main
           new Main with Act[F] {
+
+            @volatile var stopped = false
```

Comment:
is speed the only benefit of `@volatile var`? I wonder, why not use `Ref` as the code is effectful

Reply:
Well, the code is not effectful in actor's `postStop()`:
```
      override def postStop(): Unit = {
        act.value.postStop()
        ...
      }
```

Reply:
right, and we cannot use `unsafeRunSync` as glue here because the library is not built around `IO`